ElvaRivle/Misc/BugFixes

ElvaRivle opened the pull request:
ElvaRivle/Misc/BugFixes

--------------------------------------------------------------------
 MAIN
 * fixed all bugs that we discussed last time
--------------------------------------------------------------------
 OTHER
 * /
--------------------------------------------------------------------
 TODO
 * ~~logout functionality~~

evlahovlja1 committed:
ElvaRivle/Misc/BugFixes

--------------------------------------------------------------------
 MAIN
 * fixed all bugs that we discussed last time
--------------------------------------------------------------------
 OTHER
 * /
--------------------------------------------------------------------
 TODO
 * logout functionality

ElvaRivle committed:
ElvaRivle/Misc/BugFixes

--------------------------------------------------------------------
 MAIN
 * fixed all bugs that we discussed last time
--------------------------------------------------------------------
 OTHER
 * /
--------------------------------------------------------------------
 TODO
 * ~~logout functionality~~

ElvaRivle committed:
ElvaRivle/Misc/BugFixes

--------------------------------------------------------------------
 MAIN
 * .env.example added
 * typo in variable namings
 * API_URL again loaded from .env
--------------------------------------------------------------------
 OTHER
 * /
--------------------------------------------------------------------

AntMarkoLovric commented on src/components/sidebar/Sidebar.js:
```diff
@@ -42,7 +44,16 @@ const Sidebar = ({actions}) => {
+                        menu={{items: UserInfoOptions}}
+                        trigger={['click']}
+                    >
+                        <a onClick={(e) => e.preventDefault()}>
```
move function outside of jsx

becarselman replied:
you are resolve comment but don't fix anything

AntMarkoLovric commented on src/routing/protectedRoute.js:
```diff
@@ -13,7 +12,9 @@ const ProtectedRoute = () => {
       <>
         <Sidebar />
-        <Outlet />
+        <div style={{width: "100%"}}>
```
please don't use inline style

becarselman replied:
same

AntMarkoLovric commented on src/routing/publicRoute.js:
```diff
@@ -0,0 +1,22 @@
+    return !token || token === "" ?
+        (
+            <>
+                <div style={{width: "100%"}}>
```
same as the comment before regarding inline style

becarselman replied:
same

AntMarkoLovric commented on src/routing/publicRoute.js:
```diff
@@ -0,0 +1,22 @@
+    //here is the core logic that prevents unauthorized access to routes
+    //each protected route should have this component as parent
+
+    let token = localStorage.getItem("accessToken")
```
what happens if you get invalid token

ElvaRivle committed:
ElvaRivle/Misc/BugFixes

--------------------------------------------------------------------
 MAIN
 * project modal fixed action names
 * fixed PR review
--------------------------------------------------------------------
 OTHER
 * /
